.gitignore

```diff
@@ -7,54 +7,29 @@ __pycache__/
 .vscode/
 .vs/
 .idea/
-CMakeLists.txt
-cmake-build-debug
-venv/
-.venv/
 
 # Project-specific ignores
 build/
 expected/
-notes/
-baserom/
-baseroms/*/segments/
 docs/doxygen/
-*.elf
-*.sra
 *.z64
 *.n64
 *.v64
-*.map
-*.dump
-out.txt
-*.ram
 
-# Tool artifacts
-tools/mipspro7.2_compiler/
-tools/overlayhelpers/batchdisasm/output/*
-tools/overlayhelpers/batchdisasm/output2/*
-tools/overlayhelpers/batchdisasm/mipsdisasm/*
+# Tools
+.venv/
 tools/disasm/output/*
-tools/asmsplitter/asm/*
-tools/asmsplitter/c/*
 ctx.c
 tools/*dSYM/
 graphs/
 
 # Assets
-*.png
-*.jpg
-*.mdli
-*.anmi
-*.obj
-*.mtl
-*.fbx
-!*_custom*
 .extracted-assets.json
```

Comment:
Is `.extracted-assets.json` kept for convenience because it's mostly harmless?

Reply:
Oh I forgot this became extracted/VERSION/assets_extraction_times.json yeah I'll remove that too